extensions/ql-vscode/src/remote-queries/view/StarCount.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { StarIcon } from '@primer/octicons-react';
+
+type Props = { starCount?: number };
+
+const StarCount = ({ starCount }: Props) => (
+  Number.isFinite(starCount) ? (
+    <>
+      <span className="vscode-codeql__analysis-star">
+        <StarIcon size={16} />
+      </span>
+      <span className='vscode-codeql__analysis-count'>
+        {displayStars(starCount!)}
+      </span>
+    </>
+  ) : (
+    <>
```

Comment:
It feels a bit wrong to have empty spans. Can we use a `Box` component to get the layout we want?

Reply:
It was the easiest way to ensure the layouts in columns. I'll see if I can use `Box` to make it cleaner.